doc/generic/pgf/text-en/pgfmanual-en-tikz-shapes.tex

```diff
@@ -1708,11 +1708,11 @@ \subsubsection{Advanced Placement Options}
 \end{key}
 
 
-\subsubsection{Advanced Arrangements of  Nodes}
+\subsubsection{Advanced Arrangements of Nodes}
 
 The simple |above| and |right| options may not always suffice for arranging a
 large number of nodes. For such situations \tikzname\ offers libraries that
-make positioning easier: The |graphdrawing| library and the |matrix| library.
+make positioning easier: The |matrix| library and the |graphdrawing| library.
```

Comment:
What's the point of this change?

Reply:
In English after a colon one continues lower-case. I'd also avoid doubling the word "library" but write

> make positioning easier: the |matrix| and the |graphdrawing| libraries.

I do not have any preference at concerning the ordering.

Reply:
@hmenke, the point here is the next sentence these libraries are referenced. But in that sentence the order was switched. Now the first mentioned library in the above sentence corresponds to the first reference in the following sentence.